src/work-allocation/components/task-assignment/task-assignment.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { Router } from '@angular/router';
+import c = require('config');
+import { UserInfo } from 'src/app/models/user-details.model';
```

Comment:
This import and the one below shouldn't begin with `src/`.

Reply:
Ok, will change